posthog/hogql/printer.py

```diff
@@ -380,6 +380,9 @@ def visit_select_query(self, node: ast.SelectQuery):
                         else:
                             # Non-unique hidden alias. Skip.
                             column = column.expr
+                    elif isinstance(column, ast.Call):
+                        alias = print_prepared_ast(column, self.context, dialect="hogql")
```

Comment:
We're explicitly using `hogql` dialect when printing to either hogql or clickhouse so that we get consistent namings across the view and querying the view